From be17223dfe2b717322f878fbfa9e354be73f1569 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Tue, 19 Mar 2013 07:37:43 -0400
Subject: Disable unsupported audio devices

Bugzilla: 921974

Only AC97 and Intel HDA are supported audio devices. Limit build only to these devices.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index 87334a4..a5dee5d 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -10,7 +10,6 @@ CONFIG_PCNET_PCI=y
 CONFIG_PCNET_COMMON=y
 CONFIG_AC97=y
 CONFIG_HDA=y
-CONFIG_ES1370=y
 CONFIG_RTL8139_PCI=y
 CONFIG_E1000_PCI=y
 CONFIG_VMXNET3_PCI=y
diff --git a/default-configs/sound.mak b/default-configs/sound.mak
index 4f22c34..e69de29 100644
--- a/default-configs/sound.mak
+++ b/default-configs/sound.mak
@@ -1,4 +0,0 @@
-CONFIG_SB16=y
-CONFIG_ADLIB=y
-CONFIG_GUS=y
-CONFIG_CS4231A=y
